### DISCUSSION TITLE
请问flink读doris时 bitmap用什么方式可以获取到

### DISCUSSION THREAD

**Discussion by qiaohanqing** [apache/doris#19527] [A - General / Q&A]
请问flink读doris时 bitmap用什么方式可以获取到？没有找到合适的数据类型，有了解的吗，可以帮忙解答一下吗谢谢！！

**Comment by LemonLiTree** [comment]
你是要bitmap里面的存的id还是bitmap值？

**Reply by qiaohanqing** [reply]
bitmap最好，里面存的id也可以

**Reply by yagagagaga** [reply]
Here is the sample [reading bitmap using java](https://github.com/apache/doris/blob/master/samples/read_bitmap/java/ReadBitmap.java).
